### PR TITLE
Cleanup blending

### PR DESCRIPTION
**Pull request by delippi**
## DESCRIPTION OF CHANGES: 

- Moved blending python scripts from the scripts directory to the ush directory.
- Added some additional logic as suggested by Ming for the cldanl spinup step.

### Machines/Platforms:

- WCOSS2
  - [ ] Cactus/Dogwood
  - [ ] Acorn
- RDHPCS
  - [ ] Hera
  - [ ] Jet
  - [ ] Orion
  - [ ] Hercules

### Test cases: 

- [ ] Engineering tests
  - [ ] Non-DA engineering test
  - [ ] DA engineering test
    - [ ] Retro
    - [ ] Ensemble
    - [ ] Parallel
- [ ] RRFS fire weather
- [ ] RRFS_A:
- [ ] RRFS_B:
- [ ] RTMA:
- [ ] Others:

## ISSUE: 

- Fixes the issue(s) mentioned in #108

### CODE REVIEW

File: scripts/exrrfs_blend_ics.sh
Comment (MatthewPyle-NOAA) [inline scripts/exrrfs_blend_ics.sh:1]: Another small cleanup item would be to not copy the blending_fv3.py script into the run directory, but instead to execute it directly:

python $USHdir/blending_fv3.py . . .

File: scripts/exrrfs_make_ics.sh
Comment (MatthewPyle-NOAA) [inline scripts/exrrfs_make_ics.sh:1]: Same suggestion to directly execute the chgres_cold2fv3.py script directly from $USHdir/ if possible.

**Review by MatthewPyle-NOAA — COMMENTED** [review]
One small suggestion in how the python scripts are executed, but otherwise looks good.

**Comment by MatthewPyle-NOAA** [comment]
@delippi - here is the relevant part of the implementation standards document: "Model scripts and executables should be called explicitly, eg, $USHmodel/scriptx. "

**Review by hu5970 — APPROVED** [review]
Looks good. 
Agree with Matt

**Comment by delippi** [comment]
Hi @MatthewPyle-NOAA @hu5970 , I was able to get that to work. I had to add the LIB64dir to PYTHONPATH since now the python script and the *.so files were no longer in the same directory. I also removed the linking of those *.so files to the run directory as well.

**Review by MatthewPyle-NOAA — APPROVED** [review]
Looks good - thanks for addressing my comments!

**Pull request merged**